File: evals/fixtures/laravel/app/Services/OrderService.php
<?php

declare(strict_types=1);

namespace App\Services;

class OrderService
{
    /** @param list<array{int, int}> $lines [cents, quantity] pairs */
    public function orderTotalLabel(array $lines): string
    {
        $total = array_sum(array_map(fn (array $line): int => $line[0] * $line[1], $lines));

        return 'Order total: ' . $this->formatMoney($total);
    }

    private function formatMoney(int $cents): string
    {
        return '$' . number_format($cents / 100, 2);
    }
}

File: evals/fixtures/dotnet/src/Billing/OrderService.cs
namespace Billing;

public sealed class OrderService
{
    public string OrderTotalLabel(IEnumerable<(long Cents, int Quantity)> lines)
    {
        var total = lines.Sum(line => line.Cents * line.Quantity);
        return $"Order total: {FormatMoney(total)}";
    }

    private static string FormatMoney(long cents) => $"${cents / 100}.{cents % 100:00}";
}